Add GoogleBtn componenet

mila-mamat opened the pull request.

mila-mamat committed:
Add GoogleBtn componenet

bonnieli commented on client/src/components/GoogleBtn.js:
```diff
@@ -0,0 +1,75 @@
+
+
+
+const CLIENT_ID = '871373961261-rjej65g97dc3o6jiuflq6s2gp5v9ptut.apps.googleusercontent.com';
```
put in env var

bonnieli commented on client/src/components/GoogleBtn.js:
```diff
@@ -0,0 +1,75 @@
+const CLIENT_ID = '871373961261-rjej65g97dc3o6jiuflq6s2gp5v9ptut.apps.googleusercontent.com';
+
+
+class GoogleBtn extends Component {
```
make this into a functional component

mila-mamat committed:
update googleBtn

.changed to functional componenets
. seperated login and logout btn
. added client ID to env, made api route, but not working for react-google-login, commented out

mila-mamat committed:
Merge branch 'dev' into Mila-google-auth-btn

mila-mamat committed:
remove clientID from env

mila-mamat committed:
Merge branch 'Mila-google-auth-btn' of https://github.com/hatchways/team-lavender into Mila-google-auth-btn

mila-mamat committed:
updated file name, deleted googleBtn

changed from Login to loginBtn

mila-mamat committed:
Update App.js

mila-mamat committed:
Update App.js

mila-mamat committed:
Update App.js

mila-mamat committed:
Update .env

mila-mamat committed:
Update .env

mila-mamat committed:
Update index.js

bonnieli reviewed (approved):
In the future, we probably want to move `clientId` into a `config.js` (or something of similar) so that you can just refer to one spot if the clientId ever changes :) for now, i think this is okay
